tpke/src/lib.rs

```diff
@@ -140,7 +136,7 @@ pub fn setup<E: PairingEngine>(
     // F_0 - The commitment to the constant term, and is the public key output Y from PVDKG
     // TODO: It seems like the rest of the F_i are not computed?
     let pubkey = g.mul(x);
-    let privkey = h.mul(x);
+    let privkey = h.mul(x); // ek_i in PVSS?
```

Comment:
```suggestion
    let privkey = h.mul(x);
```
No, this is the private key share, which is different from the protocol-level "private key" (which can be very confusing). Let's call `ek_i` the (private) blinding key.